env.j2

```diff
@@ -0,0 +1,14 @@
+# Flask server configuration.
+FLASK_APP=todo_app/app
+FLASK_ENV=production
+
+# Change the following values for local development.
+SECRET_KEY=secret-key
+
+#API token and key values
+API_KEY=
+API_TOKEN=
```

Comment:
At the moment this seems to be the gap in your app - I can see it's up and running but crashing because these variables aren't set. To resolve that, we'll need to add some placeholders, like:
```jinja2
API_KEY={{ api_key }}
API_TOKEN={{ api_token }}
```
And then add [those as prompts](https://docs.ansible.com/ansible/latest/playbook_guide/playbooks_prompts.html) into your playbook so that they aren't being stored within the code